.github/workflows/test-integration.yml

```diff
@@ -29,8 +29,16 @@ jobs:
       matrix:
         test-path: ${{fromJson(needs.pre-test.outputs.matrix)}}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.1.0
+      - uses: technote-space/get-diff-action@v6.1.1
+        with:
+          PATTERNS: |
+            **/*.go
+            go.mod
```

Comment:
`**/go.mod` ?

Reply:
We only have one `go.mod` file in the repo and it is in the root dir